src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -305,7 +305,12 @@ private async Task GetTableChangesAsync(SqlConnection connection, CancellationTo
 
                             getChangesDurationMs = commandSw.ElapsedMilliseconds;
                         }
-
+                        // Get the number of lease locked rows and log them if > 0.
```

Comment:
```suggestion
                        // Also get the number of rows that currently have lease locks on them
                        // This can help with supportability by allowing a customer to see when a 
                        // trigger was processed successfully but returned fewer rows than expected
                        // because of the rows being locked.
```